src/components/Capture/capture.js

```diff
@@ -97,11 +98,15 @@ class Capture extends Component {
       return
     }
     payload = {...payload, documentType}
-    if (this.props.side === 'back' && !this.props.useWebcam) {
-      payload = {...payload, valid: true}
+    if (this.props.useWebcam) {
+      postToServer(this.createJSONPayload(payload), serverUrl, token,
+        (response) => this.onServerResponse(response),
+        (response) => this.onServerError(response)
+      )
+      this.setState({allowApiAdvancedValidation: false})
     }
     else {
-      postToServer(this.createJSONPayload(payload), serverUrl, token, (response) => this.onServerResponse(response), (response) => this.onServerError(response))
+      payload = {...payload, valid: true}
```

Comment:
The concept of valid captures is not really applicable anymore, captures should probably be marked as confirmable when they are suitable for confirmation from the user and ready to be sent to API. This requires quite a big refactoring on core therefore I won't be able to handle it as part of this story.